frontend/src/concepts/pipelines/NoPipelineServer.tsx

```diff
@@ -18,7 +18,7 @@ const NoPipelineServer: React.FC<NoPipelineServerProps> = ({
 
   return (
     <EmptyDetailsView
-      title="Start by creating a pipeline server"
+      title="Start by creating a pipeline"
```

Comment:
I'm confused. This component is named `NoPipelineServer`.
This component is shown when there is no pipeline server:
![image](https://github.com/opendatahub-io/odh-dashboard/assets/14068621/005f8b1b-a8d9-412e-9e38-2321391a5206)

Also shown when there is a pipeline server but no pipelines. On this screen, the `Create pipeline` button opens the configure pipeline server model which isn't right to me.
![image](https://github.com/opendatahub-io/odh-dashboard/assets/14068621/b3e8367b-2f91-431d-9556-5b591a2b2771)